config/nrfconnect/chip-module/Kconfig

```diff
@@ -397,6 +397,33 @@ config CHIP_CRYPTO_PSA_DAC_PRIV_KEY_KMU_ENCRYPTED
 
 endif # CHIP_CRYPTO_PSA_MIGRATE_DAC_PRIV_KEY
 
+config CHIP_STORE_KEYS_IN_KMU
+	bool "Use KMU to store crypto keys"
+	default y
```

Comment:
I think this should be default configuration at some point of time, not sure if we should enable it right away without thorough testing. Also, for now, i would limit it to nRF54L, not sure if H won't enable automatically?

Reply:
There is no KMU in H. :) This feature is limited to nRF54L only.

Reply:
I will remove `default y` before merging it.